apps/admin-server/src/lib/hasRole.ts

```diff
@@ -0,0 +1,20 @@
+import { roles } from './roles';
+
+interface user {
+  [key:string]: string | number,
+}
```

Comment:
Je wilt checken of de user een role heeft, en hoewel dit stukje technisch correct is, geeft het niet duidelijk weer wat je wilt. Het lijkt me duidelijker als je een interface (`MightHaveRole` bijv) maakt waarin je een `role?: string` op een object definieert.

Reply:
Yes, veel mooier